src/Input/Mouse.cpp

```diff
@@ -125,39 +136,35 @@ void Mouse::_setType(Cursor state)
             break;
         case Cursor::SCROLL_W:
             _ui = new UI::Image("art/intrface/scrwest.frm");
-            _ui->setYOffset( -_ui->height()*0.5);
+            _ui->setOffset(0, -_ui->size().height() / 2);
```

Comment:
Do you have a better idea? In this case we are setting the absolute position of Image, it is obvious since it right after the constructor.
In case you need to leave other coordinate unchanged, you will have to do:
_ui->setOffset(_ui->offset().x(), .... );